source/js/components/petition/petition.jsx

```diff
@@ -275,9 +275,6 @@ export default class Petition extends React.Component {
     // validate data here. Do not continue unless we're cool.
     let hasName = this.givenNames.element.value && this.surname.element.value;
     let email = this.email.element.value;
-
-    email = email && this.validatesAsEmail(email);
```

Comment:
I removed this line as we were performing logically "AND" between email and validatesAsEmail(), now as we are removing this function, so we have to remove this too.